packages/swapper/src/swappers/thorchain/utils/makeSwapMemo/makeSwapMemo.ts

```diff
@@ -26,7 +26,12 @@ export const makeSwapMemo = ({
       details: { buyAssetId }
     })
 
-  const memo = `s:${thorId}:${destinationAddress}:${limit}:${THORCHAIN_AFFILIATE_NAME}:${THORCHAIN_AFFILIATE_BIPS}`
+  // bch hack
```

Comment:
Can we explain the hack, similar to what you did in the PR description body?
This doesn't say much, we're removing the `bitcoincash:` prefix but why? We should always aim for full context, the next person reading this would have no idea why this is